test/dummy/SonarSensor.cpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file SonarSensor.cpp
+ * @brief ソナーセンサクラスで用いる関数（ダミー）
+ * @author YKhm20020
+ */
+
+#include "SonarSensor.h"
+using namespace ev3api;
+
+// コンストラクタ
+SonarSensor::SonarSensor(ePortS port) : distance(100) {}
+
+// RGB値を取得
```

Comment:
```suggestion
// センサーまでの距離を取得
```
RGB値じゃなくて、センサーまでの距離じゃない？